internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -264,33 +295,31 @@ func (opts *Opts) setupCloseHandler() {
 	}()
 }
 
-// askDBUserAccessListOptions allows the user to set required flags by using interactive prompts
-func (opts *Opts) askDBUserAccessListOptions() error {
+// askDBUserOptions allows the user to set required flags by using interactive prompts
+func (opts *Opts) askDBUserOptions() error {
 	var qs []*survey.Question
 
-	message := "Insert the Username for authenticating to MongoDB"
+	message := ""
 	dbUser := opts.DBUsername
 	if dbUser == "" {
 		dbUser = dbUsername()
 		if dbUser != "" {
-			message = fmt.Sprintf("Insert the Username for authenticating to MongoDB [Press Enter to use '%s']", dbUser)
+			message = fmt.Sprintf("Press Enter to use '%s']:", dbUser)
```

Comment:
sorry of this ha snot being clear but thsi is what I see
no default will print
`Database User Username :`
If you set a default
`Database User Username Press Enter to use 'something']::`

You still have the double colon(:) problem, also for this particular case it looks like you forgot the starting `[`